### PR TITLE
Slider: Touch events added to support mobile devices

### PR DESCRIPTION
**Pull request by vladislav-sevostyanov**

### CODE REVIEW

File: packages/slider/src/button.vue [hunk 1]
```diff
@@ -130,7 +131,9 @@
         event.preventDefault();
         this.onDragStart(event);
         window.addEventListener('mousemove', this.onDragging);
+        window.addEventListener('touchmove', this.onDragging);
         window.addEventListener('mouseup', this.onDragEnd);
+        window.addEventListener('touchend', this.onDragEnd);
```
Comment (syn-zeta) [inline packages/slider/src/button.vue]: The listeners should be removed `onDragEnd` as well.

File: packages/slider/src/button.vue [hunk 2]
```diff
@@ -189,6 +200,7 @@
           }, 0);
           window.removeEventListener('mousemove', this.onDragging);
           window.removeEventListener('mouseup', this.onDragEnd);
+          window.removeEventListener('touchend', this.onDragEnd);
```
Comment (syn-zeta) [inline packages/slider/src/button.vue]: Please remove `touchmove` as well, added on line 134 above.

**Comment by vladislav-sevostyanov** [comment]
It's done, comrade.

**Comment by wacky6** [comment]
So element is now accepting mobile optimization PRs?

**Comment by syn-zeta** [comment]
Is touch support considered mobile only? Most manufacturers offer at least a couple laptop/all-in-one touch options.

**Comment by wacky6** [comment]
Never had an all-in-one, not sure whether browsers in all-in-one devices emit touch events.
If they don't, for me, touch events belong to mobile / tablet optimization.

**Comment by syn-zeta** [comment]
Yes, laptops/desktops with touchscreen emit touch events.

**Comment by wacky6** [comment]
Then sounds like element-ui is going to be touch-friendly and accidentally become usable in touch-only environments. :)

**Comment by Leopoldthecoder** [comment]
My concern is that we don't want the library to be bloated with mobile-devide-only code. But this PR has only 13 lines of additions, so I merged it.

**Pull request merged**